docs/Windows.md

```diff
@@ -1,43 +1,172 @@
+
+Then execute the following four commands from within the Native Tools Command Prompt:
 
-1. Extract the toolchain artifact.  We assume that the toolchain will be installed to `C:\Library`.  Extracting it to that location should give you a hierarchy that looks like `C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin\swiftc.exe`.
-2. In order to develop with this toolchain, you will need an installation of the Windows SDK.  The easiest way to do this is to install Visual Studio.  Additionally, you will need to copy a few files into the SDK to make it usable from swift.
 ```cmd
 curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/ucrt.modulemap" -o "%UniversalCRTSdkDir%\Include\%UCRTVersion%\ucrt\module.modulemap"
 curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/visualc.modulemap" -o "%VCToolsInstallDir%\include\module.modulemap"
 curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/visualc.apinotes" -o "%VCToolsInstallDir%\include\visualc.apinotes"
 curl -L "https://raw.githubusercontent.com/apple/swift/master/stdlib/public/Platform/winsdk.modulemap" -o "%UniversalCRTSdkDir%\Include\%UCRTVersion%\um\module.modulemap"
 ```
-3. You will need to add the ICU libraries for the target.  The nightlies are built against ICU 64.2 from the ICU project.  You can download the binaries for that from http://download.icu-project.org/files/icu4c/64.2/icu4c-64_2-Win64-MSVC2017.zip.
+
+Close this instance of the Native Tools Command Prompt after those commands have been successfully executed.
+
+#### Swift Toolchain & Platform SDK concepts
+
+The installation instructions that follow will result in a directory tree that has a well thought-out structure. To understand this directory structure better you may consult  <https://github.com/compnerd/windows-swift/blob/master/docs/details.md>. But first, you might want to continue with the installation.
+
+#### Downloading the nightlies
+
+1. Go to  <https://dev.azure.com/compnerd/windows-swift>.
+2. Choose `Pipelines` > `Pipelines` from the left of the dashboard.
+3. Choose 'Runs'.
+4. Use the filter symbol to search for "VS2017".
+5. In the results list, scroll down until you see the first successful build (with a green OK symbol) and click on it.
+6. Under "Artifacts", click on the "... published" link.
+7. Download windows-toolchain-amd64.msi, windows-sdk.msi, and windows-runtime-amd64.msi by clicking on the appropriate down-arrows on the right. Be sure to really download these files from the same build (i.e. do not switch the build for the next download, and be careful when updating)! These files will be downloaded as zip files. Unless they are not automatically unzipped during the download process, unzip them to obtain *.msi files in the extracted directories.
```

Comment:
I think that the `*` needs to be quoted, as it is being processed as the beginning of an italics block.

If you are so motivated, all of this is possible via the REST API that azure exposes.  Perhaps you would write up a powershell cmdlet for this?  See the `Swift-DownloadArtifacts` cmdlet in Utiltiies.